crates/krun-guest/src/pulse.rs

```diff
@@ -1,14 +1,17 @@
 use std::{fs, path::PathBuf, process::Command};
 
 use anyhow::{Context, Result};
-use utils::env::find_in_path;
+use utils::{env::find_in_path, stdout::create_stdout_stderr};
 
 pub fn setup_pulse_proxy(run_path: PathBuf) -> Result<()> {
     if let Some(socat_path) = find_in_path("socat")? {
         let pulse_path = run_path.join("pulse");
         fs::create_dir(&pulse_path)
             .context("Failed to create `pulse` directory in `XDG_RUNTIME_DIR`")?;
         let pulse_path = pulse_path.join("native");
+
+        let (stdout, stderr) = create_stdout_stderr("socat")?;
```

Comment:
WDYT about just using [`piped`](https://doc.rust-lang.org/std/process/struct.Stdio.html#method.piped)? Or is this important for debuggability?

Reply:
I think it could be useful in case something goes wrong with the pulseaudio bridge.